src/components/Map.js

```diff
@@ -98,10 +99,24 @@ export default class Map extends Component {
           )}
         </MapView>
         <Input
+          lock={this.props.lock}
           getMessages={this.props.getMessages}
           location={this.props.location}
           userAuth={this.props.userAuth}
           login={this.props.login}
+          username={this.props.username}
+        />
+        <Prompt
```

Comment:
Prompt requires a `title` attribute, otherwise, the simulator will keep giving errors.